anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -487,5 +517,8 @@ def transform(self, frames, show_progress=False):
         )
 
         features = contract_pairwise_feat(pairwise_ellip_feat, species)
-
-        return features
+        normalized_features = normalize_basis(self.radial_basis, features)
+        if normalize:
+            return normalized_features
+        else:
+            return features
```

Comment:
Lol so if we don't want to normalize, we compute it anyways? See comment above about optional normalization, but if normalize=False, we shouldn't compute the normalization

Reply:
Fixed this long ago